src/api/route-services/access.js

```diff
@@ -0,0 +1,104 @@
+
+const _ = require('lodash');
+
+const config = require('../../config');
+const AWS = require('../../utils/requireAWS');
+const getLogger = require('../../utils/getLogger');
+
+const {
+  createDynamoDbInstance,
+  convertToJsObject,
+  convertToDynamoDbRecord,
+} = require('../../utils/dynamoDb');
+const { isRoleAuthorized } = require('./roles');
+
+
+const logger = getLogger('[AccessService] - ');
+
+class AccessService {
+  constructor() {
+    this.userAccessTableName = `user-access-${config.clusterEnv}`;
+    this.inviteAccessTableName = `invite-access-${config.clusterEnv}`;
```

Comment:
better to put the particular database names in default-config and import the names from there

Reply:
I'll create a ticket to move the names into a default-config